buildSrc/build.gradle.kts

```diff
@@ -71,13 +71,7 @@ subprojects {
 
         apply(from = "../../../gradle/shared-with-buildSrc/code-quality-configuration.gradle.kts")
     }
-
-    apply(plugin = "idea")
     apply(plugin = "eclipse")
-
-    configure<IdeaModel> {
-        module.name = "buildSrc-${this@subprojects.name}"
-    }
```

Comment:
This in no longer needed. IntelliJ automatically names the modules `buildSrc.<project-name>`.